garak/generators/mistral.py

```diff
@@ -20,6 +21,45 @@ class MistralGenerator(Generator):
     ENV_VAR = "MISTRAL_API_KEY"
     DEFAULT_PARAMS = Generator.DEFAULT_PARAMS | {
         "name": "mistral-large-latest",
+        # Sampling parameters
+        "top_p": None,                    # Nucleus sampling (0.0-1.0)
+        "frequency_penalty": None,        # Penalize word repetition by frequency (0.0-2.0)
+        "presence_penalty": None,         # Penalize word repetition (0.0-2.0)
+        # Generation control
+        "n": None,                        # Number of completions to return
+        "stop": None,                     # Stop sequences (string or list)
```

Comment:
Prefer consistency with other `stop` params and require a `list[str]`:
```suggestion
        "stop": [],                     # Stop sequences (list[str])
```